### PR TITLE
[ on-call ] Updating logging on add user to context middleware to provide more information

### PR DESCRIPTION
**Pull request by mmayo-truss**
## Summary

Adding more information to a logging statement for future debugging purposes.

### CODE REVIEW

**Review by kctruss — APPROVED** [review]
👍

File: pkg/handlers/authentication/add_user_to_context.go [hunk 1]
```diff
@@ -18,7 +20,7 @@ func AddAuditUserToRequestContextMiddleware(appCtx appcontext.AppContext) func(n
 			if newAppCtx.Session() != nil {
 				auditUser, err := user.NewUserFetcher(query.NewQueryBuilder()).FetchUser(newAppCtx, []services.QueryFilter{query.NewQueryFilter("id", "=", newAppCtx.Session().UserID)})
 				if err != nil {
-					newAppCtx.Logger().Info("No user attached to the session")
+					newAppCtx.Logger().Error("Error encountered when fetching user with session UserID.", zap.Error(err))
```
Comment (duncan-truss) [inline pkg/handlers/authentication/add_user_to_context.go:23]: I think we can see the SessionId value in the logs already but could you also add the UserID value here either to the message or as `zap.String("userId", newAppCtx.Session().UserID)`.

**Review by duncan-truss — APPROVED** [review]

File: pkg/handlers/authentication/add_user_to_context.go (hunk 1 above)
Comment (mmayo-truss) [reply pkg/handlers/authentication/add_user_to_context.go]: Sounds good. I'll add that.

**Review by afoon — APPROVED** [review]
LGTM! Error showed as expected with the userId

**Pull request merged**